README.md

```diff
@@ -23,12 +23,12 @@ wget https://raw.githubusercontent.com/SIMPLE-AstroDB/SIMPLE-binary/main/SIMPLE.
 
 ### Running
 Then run the application with   
-```python 
-python simple_app/app_simple.py
+```bash 
+python -m simple_app.app_simple
```

Comment:
I believe this means the .devcontainer/devcontainer.json file also needs to be updated to add `-m`. It didn't run properly when I tried to run via Codespaces otherwise.